### ISSUE TITLE
Redefining Arctic and North Atlantic subpolar regions

### ISSUE THREAD

**Issue by milenaveneziani** [MPAS-Dev/geometric_features#118]
I see the following two main issues with the current definition of 'Arctic' and sub Arctic regions:
1) the current `Arctic` geojson feature is a weird region that covers most of the Arctic, but not quite all of it (excluding coastal seas such the Beaufort Sea, Chukchi Sea, etc);
2) the subpolar North Atlantic regions are defined weirdly sometimes, perhaps more in line with geographical definitions rather than oceanographic ones: one example is the Lab Sea, which is a small triangle, south of a `DavisStrait` feature..

Below, I am suggesting a possible redefinition. Please let me know if you have an opinion on this, and your suggestions, so that I can go ahead and make a PR to address it.

**Comment by milenaveneziani** [comment]
**Arctic**

Currently, what we think of for 'Arctic' consists of the following geojson features:
1) Arctic
2) Barentsz (misspelled: should be simply `Barents`)
3) Kara Sea
4) White Sea
5) Laptev Sea
6) East Siberian Sea
7) Chukchi Sea
8) Beaufort Sea (*very* different from typical definition of Beaufort Gyre)
9) Lincoln Sea
10) and then there is a piece of the current Greenland Sea that should go in `Arctic`, north of Fram Strait (79N).

My first suggestion is to combine 1 and 5-10 above into a new `Arctic` feature.
My second suggestion is to combine 2-4 into a Barents-Kara Seas feature (this is because sometimes these two 'entrance' seas are considered separately from the main Arctic).
My third suggestion is to add new Arctic subregions, the details of which I still need to figure out, but they would approximately be the following:
- a Beaufort gyre (defined as from Proshutinsky's papers: 130W-170W, 70.5N-80.5N and deeper than 300m);
- a Eurasian Basin (deeper than ~1000m)
- a Amerasian Basin, excluding the Beaufort gyre piece (again deeper than ~1000m)
- a shelf region (shallower than ~1000m, expect for the BG piece again).

**Comment by milenaveneziani** [comment]
# Subpolar North Atlantic

Here is what we have now:
1) Greenland Sea (mostly OK, but extends north of Fram Strait). Includes both Greenland and Iceland Seas
2) Norwegian Sea. Includes both Norwegian and Lofoten Basins
3) Baffin Bay
4) Davis Strait
5) Labrador Sea (different from conventional Lab Sea)
6) Northwestern Passages (consistent of the whole Canadian Archipelago plus Foxe Basin, which to me would make more sense to include into a Hudson Bay region)
7) Hudson Bay
8) Hudson Strait

My first suggestion here is to slightly adjust 1 to cut it at 79N (which is also our definition of Fram Strait transect). The remaining northern piece will go into the Arctic feature (see Arctic box above).

My second suggestion is perhaps to leave 3 as is, but combine 4 and 5 (and maybe readjust) to make a new 'Labrador Sea' feature.

My third suggestion is to remove Foxe Basin from 6, and perhaps rename the feature to 'Canadian Archipelago'.

My fourth suggestion is to combine 7 and 8 and Foxe Basin into a new 'Hudson Bay' feature.

**Comment by xylar** [comment]
@milenaveneziani, I don't see a problem with these redefinitions as long as you document your process by putting the script you use to create the new features in
https://github.com/MPAS-Dev/geometric_features/tree/master/feature_creation_scripts
along with your PR(s).

The existing definitions are from:
http://www.marineregions.org/sources.php#iho
Yes, possibly more geographic than oceanographic.

Let me know if you need help with using the new geometric features to do this.  I think it should be a relatively straightforward merge followed by a split.  Please follow the instructions in the README.md on how to put features back in `geometric_data`.  You'll need to manually `git rm` the features you don't want anymore.  After updating, adding and/or removing features, update the database of features and tags.

**Comment by milenaveneziani** [comment]
These are high level suggestions, I haven't included any maps, but please let me know if you'd like to see some specific maps of the current features for reference.
Pinging @proteanplanet (Andrew, I hope :), @maltrud, and @lconlon in case they don't see this and would like to comment.

**Comment by milenaveneziani** [comment]
Thanks @xylar: I will follow the README instructions better this time around.

**Comment by milenaveneziani** [comment]
One last thing I forgot to mention: we could also add a 'Irminger Sea' feature in the subpolar North Atlantic regions mentioned above.

**Comment by proteanplanet** [comment]
Following our discussion just now, we agreed:
1) The Baltic should always be open, although @xylar has that covered in COMPASS now. 
2) We should be mindful of the limnological cost in E3SM, but ideally, it should be included wherever possible.
3) The generation of regions within the Arctic should be altered from the first attached figure:
 
![RASM_ocean_mask_key](https://user-images.githubusercontent.com/14379090/67725941-bb610e00-f9a9-11e9-8520-f433fac2094e.png)

so that the central Arctic is defined in terms of the more common MASIE regions used in Arctic literature: https://nsidc.org/data/masie/browse_regions.  

4) We are investigating the use of a more accurate bathymetric dataset for the Arctic, either a latest version of IBCAO consistent with GEBCO. Further to this, @eclare108213 is currently investigating other options on our behalf to ensure we are up to date with potentially new datasets to be made available.

**Comment by xylar** [comment]
@proteanplanet et al., where are the definitions of the MASIE regions available?  Or alternatively the RASM variants on them (as they seem to be different)?  If they are available as polygons in some form, I could likely convert them to geojson format in a couple of hours and have the regions up in `geometric_features` later this week.

**Comment by proteanplanet** [comment]
Hi @xylar and @milenaveneziani:  I'm attaching here the masks I developed on the RASM grid, in case that is of use. @milenaveneziani, when you create the new shapes by combining these with MASIE, please try to preserve the central Arctic outline as best you can, as it almost perfectly matches that used by the Arctic remote sensing community, including the all-important gate across Fram Strait. 

[RASM_POPCICE_GRID_MASKS_AND_METRICS.nc.zip](https://github.com/MPAS-Dev/geometric_features/files/3784311/RASM_POPCICE_GRID_MASKS_AND_METRICS.nc.zip)

If you need me to turn any of the masks into geoshapes, let me know, as I can easily do that.

**Comment by milenaveneziani** [comment]
Hi @proteanplanet: Wieslaw did send me the RASM_POP mask file, although I think the map I have (and that you attached) plus a isobath contour will be more helpful to me (and I am hopeful I can find that information on the MASIE link).
More soon..

**Comment by xylar** [comment]
> ave been using the following command without luck:
> conda install -c conda-forge -c e3sm geometric_features=0.1.4
> am I doing something wrong?

You don't need the `e3sm` channel but that shouldn't matter.  

I was able to create a new environment with it without a problem:
```
conda create -y -n test geometric_features=0.1.4 python=3.7
```
I know it's tempting to keep your old environment around but it's a much safer practice to make new environments often (and have a recipe for them lying around somewhere) than it is to keep conda-installing package into an existing one.  Packages end up being incompatible with one another and conda doesn't always realize it, or too many upgrades are required to make the environment compatible with new packages.

**Comment by milenaveneziani** [comment]
Addressed by #123 and #125.
